sql/core/src/test/scala/org/apache/spark/sql/streaming/FileStreamSourceSuite.scala

```diff
@@ -727,6 +732,48 @@ class FileStreamSourceSuite extends FileStreamSourceTest {
       }
     }
   }
+
+
```

Comment:
This line no longer exists after merging the changes from [https://github.com/apache/spark/pull/14728] and replacing my former implementation of GC for FileStreamSource with the one in that merged PR.